relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -83,6 +85,7 @@ impl SpanTagKey {
             SpanTagKey::System => "system",
             SpanTagKey::TimeToFullDisplay => "ttfd",
             SpanTagKey::TimeToInitialDisplay => "ttid",
+            SpanTagKey::Extension => "file_extension",
```

Comment:
Do we want to keep the naming consistent and have actually `FileExtension` enum variant?

Reply:
Done.